code/obj/item/device/scanners.dm

```diff
@@ -612,13 +612,27 @@ that cannot be itched
 	name = "security RecordTrak"
 	desc = "A device used to scan in prisoners and update their security records."
 	icon_state = "recordtrak"
-	var/mode = 1
 	var/datum/db_record/active1 = null
 	var/datum/db_record/active2 = null
 	w_class = W_CLASS_NORMAL
 	item_state = "recordtrak"
 	flags = FPRINT | TABLEPASS | ONBELT | CONDUCT | EXTRADELAY
 	mats = 3
+	contextLayout = new /datum/contextLayout/experimentalcircle
```

Comment:
```suggestion
```
I don't know why this compiles but it leads to weird behavior, so just instantiate the thing in `New()`

Reply:
the context layout is made the same way as #9199 so if it causes strange behaviour that should be changed also

Reply:
Oh I didn't notice that. and I merged that!!! Probably fine then